execution_engine/src/runtime_context/mod.rs

```diff
@@ -126,6 +127,7 @@ where
             transfers,
             remaining_spending_limit,
             package_kind,
+            last_message_cost: U512::from(0),
```

Comment:
i don't think the `last_` prefix is necessary

Reply:
renamed here: https://github.com/casper-network/casper-node/commit/8171806b24ac2178a16b6721587d073eeb42a0d5